File: Documentation/5.4/Samples/python/ClientApi/Session/HowTo/GetChangeVector.py
from examples_base import ExampleBase


class Foo:
    # region get_change_vector_1
    def get_change_vector_for(self, entity: object) -> str: ...

    # endregion


class GetChangeVector(ExampleBase):
    def setUp(self):
        super().setUp()

    def test_get_change_vector(self):
        with self.embedded_server.get_document_store("GetChangeVector") as store:
            with store.open_session() as session:
                # region get_change_vector_2
                employee = session.load("employees/1-A")
                change_vector = session.advanced.get_change_vector_for(employee)
                # endregion

File: Documentation/5.4/Samples/php/ClientApi/session/HowTo/GetChangeVector.php
<?php

namespace RavenDB\Samples\ClientApi\Session\HowTo;

use Employee;
use RavenDB\Documents\DocumentStore;

interface FooInterface
{
    # region get_change_vector_1
    public function getChangeVectorFor(?object $instance): ?string;
    # endregion
}

class GetChangeVector
{
    public function example(): void
    {
        $store = new DocumentStore();
        try {
            $session = $store->openSession();
            try {
                # region get_change_vector_2
                $employee = $session->load(Employee::class, "employees/1-A");
                $changeVector = $session->advanced()->getChangeVectorFor($employee);
                # endregion
            } finally {
                $session->close();
            }
        } finally {
            $store->close();
        }
    }
}
